react.html

```diff
@@ -74,7 +74,7 @@
   	  	this.interval = setInterval(this.timerTick, 1000);
   	  },
   	  componentWillUnmount: function() {
-  	  	clearInterval(this.interval);
+  	  	clearInterval(this.state.strikes == 1000);
```

Comment:
This was correct the first time, you need to call `clearInterval` with the value returned from `setInterval`. But that's just to clean up the timer when this component unmounts. To make the timer reset after so many strikes would be a different part of the code, I'll point out in another comment.